app/views/users/index.haml

```diff
@@ -27,7 +27,7 @@
         %td= yes_no_image user.supervisor?
         %td= yes_no_image user.staff?
         %td
-          - if user.driver? && user.incident_reports.present?
+          - if user.driver?
```

Comment:
So you made the change for every user that is a driver to have an `incidents_user_url`? Regardless of if they have an incident report?